tests/util.go

```diff
@@ -39,6 +40,15 @@ const (
 	paramFlag            = "-p"
 )
 
+func CreateDirIfNotExist(dir string) {
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
+		err = os.MkdirAll(dir, 0755)
+		if err != nil {
+			panic(err)
```

Comment:
generally don't call Panic in utility funcs. If Panic must be called then consider changing the func name to something like `CreateDirIfNotExistOrDie`